### ISSUE TITLE
Upgrade chart.js library to latest 4.x

### ISSUE THREAD

**Issue by dou-du** [aiidalab/widget-bandsplot#54]
There are several issues may related to the outdate of the chart.js library.

For instance,

issue: #49 
PR: #44

**Comment by unkcpz** [comment]
I agree this is the urgent issue. Me and @superstar54 start using plotly for our results display widget. It has some very good feature. Maybe we consider to move to plotly which also have JS backend https://plotly.com/javascript/

**Comment by unkcpz** [comment]
Please make a note on the AiiDAlab meeting note and we will discuss at the meeting tomorrow. Thanks!
